lumen_runtime/src/time/system.rs

```diff
@@ -0,0 +1,34 @@
+use num_bigint::BigInt;
+
+use crate::time::convert;
+use crate::time::Milliseconds;
+use crate::time::Unit::{self, *};
+use crate::time::{
+    MICROSECONDS_PER_MILLISECOND, MILLISECONDS_PER_SECOND, NANOSECONDS_PER_MILLISECONDS,
+};
+
+cfg_if::cfg_if! {
+  if #[cfg(all(target_arch = "wasm32", feature = "time_web_sys"))] {
+     mod web_sys;
+     pub use self::web_sys::*;
+  } else {
+     mod std;
+     pub use self::std::*;
+  }
+}
+
+pub fn time(unit: Unit) -> BigInt {
+    let milliseconds = time_in_milliseconds();
+
+    match unit {
+        Second => (milliseconds / MILLISECONDS_PER_SECOND).into(),
+        Millisecond => milliseconds.into(),
+        Microsecond => (milliseconds * MICROSECONDS_PER_MILLISECOND).into(),
+        Nanosecond => (milliseconds * NANOSECONDS_PER_MILLISECONDS).into(),
+        _ => convert(
+            (milliseconds * NANOSECONDS_PER_MILLISECONDS).into(),
+            Nanosecond,
+            unit,
+        ),
+    }
```

Comment:
There is probably a way to place this in the time module for use between monotonic and system. I wasn't able to do so, but only tried for about 10 minutes this morning.

Reply:
Figured it out